src/components/Typography/Typography.js

```diff
@@ -6,7 +6,7 @@ import styles from './styles';
 
 const Typography = ({className, color, children, variant, classes}) => (
   <MaterialTypography
-    className={className ? [classes.root, className] : classes.root}
+    className={className ? [classes.root, className].join(' ') : classes.root}
```

Comment:
what does [classes.root, className].join(' ') do?

Reply:
classes.root and className are both strings, so the join makes it like "root someClassName".